extensions/acc-core.md

```diff
@@ -25,20 +25,20 @@ authority layer of the network.  Further, we believe it is helpful to client aut
 they may provide guided account creation and validation for their users, regardless of network
 configuration.
 
-## The `REG` command
+## The `ACC` command
 
-The `REG` command is the main command used by the account registration subsystem.  It provides
-several subcommands listed in the `REGCOMMANDS` ISUPPORT type.  The core sub-commands are listed
+The `ACC` command is the main command used by the account registration subsystem.  It provides
+several subcommands listed in the `ACCCOMMANDS` ISUPPORT type.  The core sub-commands are listed
 below:
 
-## The `REG CREATE` sub-command
+## The `ACC REGISTER` sub-command
```

Comment:
Each subcommand should be it's own extension rather than in the framework spec (extensions/acc/register).

This allows for a more modular system in which networks can choose which features to support.